src/bin/vip-dev-env-import-sql.js

```diff
@@ -17,6 +17,7 @@ import command from '../lib/cli/command';
 import { getEnvironmentName, handleCLIException } from '../lib/dev-environment/dev-environment-cli';
 import { exec, resolveImportPath } from '../lib/dev-environment/dev-environment-core';
 import { DEV_ENVIRONMENT_FULL_COMMAND } from '../lib/constants/dev-environment';
+import chalk from 'chalk';
```

Comment:
This should be under external dependencies.

Reply:
Fixed. Thank you!